Restructure policy creation layout

jordojordo opened the pull request:
Fix #723 

Initial restructuring of policy table layout

![policy-table](https://github.com/rancher/kubewarden-ui/assets/40806497/75b90153-a0ae-4284-a45d-3392f225e255)

kravciak commented:
- did we stop using [displayName](https://github.com/kubewarden/pod-privileged-policy/blob/main/metadata.yml#L45)?
- some policies are duplicit when I show also unofficial policies (container resources policy for example)
- I like that filter#name recognizes policy name and keywords. But filter#attribute also search keywords. I propose to change this into:
  - filter#attributes: resources (pods, ...), features (mutation, context-aware, ...), organization?
  - filter#name: name, title, keywords
- policy readme/creation is missing header with steps (it shows policy name, current step). So I don't know which policy I initially clicked

kwwii commented:
As a note, we've removed the steps in the wizard as the new design supports a simple workflow without inherint steps. As this is work in progress it might not be apparent yet.

jordojordo commented:
> did we stop using [displayName](https://github.com/kubewarden/pod-privileged-policy/blob/main/metadata.yml#L45)?

Fixed

> some policies are duplicit when I show also unofficial policies (container resources policy for example)

This is due to the organization not appearing within the table list, they aren't exact duplicates as they are from different repositories.

> - filter#attributes: resources (pods, ...), features (mutation, context-aware, ...), organization?
> - filter#name: name, title, keywords

I've updated the filtering, the attributes input no longer has the organization, I removed the keywords and added the features. As for the name input, this now searches for keywords as well as name and titles

> policy readme/creation is missing header with steps (it shows policy name, current step). So I don't know which policy I initially clicked

I replaced the policy name as the header when moving to the other steps to make it clearer.

> table does not have side borders, previous page (policy list) does have borders

Fixed
